assets/startup.sh

```diff
@@ -38,3 +38,7 @@ for f in /docker-entrypoint-initdb.d/*; do
   esac
   echo
 done
+
+echo "#########################"
+echo "DATABASE IS READY TO USE!"
+echo "#########################"
```

Comment:
We also created a a tmp file that we then wired to healthchecks:
```
# healtcheck can check for this file
echo "We're good to go"
echo date >> /tmp/db_up
```

So in our compose we just had:
```
   healthcheck:
      test: bash -c "[ -f /tmp/db_up ]"
```

Which might be convenient for others.

Reply:
```suggestion
# add a file that healtchecks can reference
echo date >> /tmp/db_up
```